Gemfile

```diff
@@ -1,12 +1,18 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.0'
 
 gem 'annotate', '~> 3.2'
 
+gem 'lefthook', '~> 1.5', '>= 1.5.2'
+gem 'rails_best_practices', '~> 1.23', '>= 1.23.2'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
 gem 'rails', '~> 6.1.7', '>= 6.1.7.6'
+gem 'reek', '~> 6.1', '>= 6.1.4'
+gem 'rubocop', '~> 1.57', '>= 1.57.1'
```

Comment:
(Except `rails` gem)
We will only use these gems during development/testing, so let's move them to the dev/test group!